src/services/dbServices.js

```diff
@@ -43,7 +48,21 @@ const DbServices = {
       updateInfo,
       options
     );
-  }
+  },
+
+  /**
+   * @param {object} options query option
+   * @param {object} Request model /table
+   * @returns {Promise} Promise resolved or rejected
+   * @description get all rows where option matches
+   */
+  findRequest(options) {
```

Comment:
Since this function `(findRequest)` is tied to the `Request model`, I think you should move it to its own service file. This file `(dbServices.js)` is a generic file for all services.

Reply:
done